<ui version="4.0" >
 <class>graphSettings</class>
 <widget class="QWidget" name="graphSettings" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>155</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>2222</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>1</number>
   </property>
   <property name="topMargin" >
    <number>1</number>
   </property>
   <property name="rightMargin" >
    <number>1</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <property name="horizontalSpacing" >
    <number>1</number>
   </property>
   <property name="verticalSpacing" >
    <number>1</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="maximumSize" >
      <size>
       <width>12222</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string>Сетка</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignLeading</set>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>1</number>
      </property>
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>1</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>1</number>
      </property>
      <item>
       <widget class="QPushButton" name="gridSwitch" >
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Вкл/Выкл сетку</string>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gridPlus" >
        <property name="maximumSize" >
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Увеличить шаг сетки</string>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="gridMinus" >
        <property name="maximumSize" >
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Уменьшить шаг сетки</string>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_3" >
     <property name="title" >
      <string>Цвета</string>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>1</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>1</number>
      </property>
      <property name="horizontalSpacing" >
       <number>1</number>
      </property>
      <property name="verticalSpacing" >
       <number>1</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Цвет фона</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="forecolor" >
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Цвет сетки</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="gridcolor" >
        <property name="autoFillBackground" >
         <bool>true</bool>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="flat" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_3" >
     <property name="text" >
      <string>Масштаб</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QSlider" name="zoomSlider" >
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <number>10000</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>167</width>
       <height>31</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
